Mechanisms_evolution/Mechanisms_Evolution.Rmd

```diff
@@ -263,11 +233,15 @@ t.test(Red1$snails, White1$snails)
 t.test(Red2$snails, White2$snails)
 ```
 
-The output from `t.test` gives you a lot of information. For this class, we are going to focus on the p-value. A p-value is the probability of getting data as extreme or more extreme than the observed data given that the null hypothesis is true.
+The output from `t.test` gives you a lot of information. For this class, we are going to focus on the p-value. A p-value is the probability of getting data different from the observed data given that the null hypothesis is true.
+
+Imagine you are willing to accept a 5% probability that you reject the null hypothesis if it is really true. That's like saying for 20 experiments where the null is true, one of them will probably appear as if the null is false (1/20 = 5%). Because you have some variance in your data you need to allow for some probability of being wrong. If your p-value is less than 0.05 you will reject the null hypothesis. If it is greater than 0.05 you cannot reject the null hypothesis. 
 
-Imagine you are willing to accept a 5% probability that you reject the null hypothesis if it is really true. That's like saying for 20 experiments where the null is true, one of them will probably appear as if the null is false (1/20 = 5%).  Because you have some variance in your data you need to allow for some probability of being wrong. If you're having trouble with the concept of variance think about flipping a coin. Most of the time your coin flips will not be 50/50 heads/tails. But you still don't reject it being a fair coin. If your p-value is less than 0.05 you will reject the null hypothesis. If it is greater than 0.05 you cannot reject the null hypothesis. 
 
-For each experiment
+## Lab 2 Report Submission
+Turn in a hard copy of your lab report that includes the following:
+1. Plot of Experiment 1 Means from Section 4d
+2. Plot of Experiment 2 Means from Section 5
+3. 
```

Comment:
does the lab report include their code with their comments and hypotheses?

Reply:
No, they only include their summary table, plots, and a short explanation:
Summary table for your section’s data
Line graph of Experiment 1 means from Step 4d
Line graph of Experiment 2 means from Step 5
Results of your t-tests in table form for both experiments from Step 6
Did you reject or fail to reject your null hypotheses? Explain why you rejected or failed to reject your hypotheses and explain the processes that influenced both experiments (4 sentences each)

Reply:
Turn in your code. Code should be organized, include good comments, and include only code that works and does not produce errors.